platform/src/main/java/platform/pageobjects/MyPage/Membership.java

```diff
@@ -0,0 +1,132 @@
+package platform.pageobjects.MyPage;
+
+import java.util.List;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.junit.Assert;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+import utils.CommonFunctions;
+
+public class Membership {
+
+	List<String[]> testdata;
+	public WebDriver driver;
+	private static Logger log = LogManager.getLogger(Membership.class.getName());
+	int waitTime = 5;
+
+	public Membership(WebDriver driver) {
+		this.driver = driver;
+		PageFactory.initElements(driver, this);
+	}
+
+	@FindBy(xpath = "//a[@class='link js-not-link']")
+	public WebElement accountInformation;
+
+	@FindBy(xpath = "//a[@class='submenu__link'][contains(text(),'会員ステータス')]")
```

Comment:
Please replace reference to contains(text()) with alternate xpath.

Reply:
@csonparote Is it ok if I use like this.
//div[@class='header__control']//li[2]//a[1]

Reply:
Yes. u can use it.

Reply:
@csonparote 
I have done necessary changes. Can you please check

Reply:
Its done

Reply:
Thank you.